src/prompt.js

```diff
@@ -1,43 +1,15 @@
 import inquirer from 'inquirer';
-import ObjectId from 'bson-objectid';
-import {
-  AWS_ACCESS_KEY_ID_LENGTH,
-  AWS_SECRET_ACCESS_KEY_LENGTH,
-} from './config';
-import initStarter from './initStarter';
-
-const validateGraaspDeveloperId = (value) => {
-  // allow valid object ids or empty
-  if (ObjectId.isValid(value) || value === '') {
-    return true;
-  }
-  return 'Graasp Developer ID is not valid. Leave it empty if you do not have it yet.';
-};
+import { validate as UUIDValidate, v4 } from 'uuid';
```

Comment:
Here did you add the functionality so that if they specify a repo, we do not ask them the Framework / Language options? Those are simply to go to the right default based on Framework / Language.